File: pickerlib/src/main/java/com/adrian/pickerlib/wheelview/OnItemSelectedRunnable.java
package com.adrian.pickerlib.wheelview;

final class OnItemSelectedRunnable implements Runnable {
    final WheelView loopView;

    OnItemSelectedRunnable(WheelView loopview) {
        loopView = loopview;
    }

    @Override
    public final void run() {
        if (loopView != null && loopView.onItemSelectedListener != null) {
            loopView.onItemSelectedListener.onItemSelected(loopView.getCurrentItem());
        }
    }
}

File: wheelviewpicker/src/main/java/com/adrian/wheelviewpicker/view/OnItemSelectedRunnable.kt
package com.adrian.wheelviewpicker.view

/**
 * author:RanQing
 * date:2018/6/25 0025
 * description:
 **/
class OnItemSelectedRunnable(val loopView: WheelView) : Runnable {
    override fun run() {
        loopView.onItemSelectedListener!!.onItemSelected(loopView.selectedItem)
    }
}